aiida/cmdline/commands/cmd_config.py

```diff
@@ -190,3 +192,30 @@ def verdi_config_caching(disabled):
                     echo.echo(identifier)
             elif disabled:
                 echo.echo(identifier)
+
+
+@verdi_config.command('upgrade')
+@click.argument('path', required=False, default=None, type=click.Path(exists=True, path_type=Path))
```

Comment:
Actually I think I will remove this command since, on calling the command, verdi is already auto-upgrading, rendering it kinda useless.
The main thing is the downgrad command